### PR TITLE
Improve the assign input on Alerts View.

### PR DESCRIPTION
**Pull request by florinbilt**

### CODE REVIEW

File: ui/perfherder/alerts/Assignee.jsx [hunk 1]
```diff
@@ -43,11 +45,13 @@ export default class Assignee extends React.Component {
   };
 
   pressedEnter = async (event) => {
-    event.preventDefault();
     if (event.key === 'Enter') {
+      event.preventDefault();
```
Comment (beatrice-acasandrei) [inline ui/perfherder/alerts/Assignee.jsx:49]: Could you quickly explain why did you move it?

File: ui/perfherder/alerts/Assignee.jsx [hunk 2]
```diff
@@ -33,7 +33,9 @@ export default class Assignee extends React.Component {
         inEditMode: true,
         // input prefills with this field, so
         // we must have it prepared
-        newAssigneeUsername: assigneeUsername,
+        newAssigneeUsername: assigneeUsername
+          ? assigneeUsername.split('/')[1]
```
Comment (beatrice-acasandrei) [inline ui/perfherder/alerts/Assignee.jsx:37]: I don't think you need to do that because on line 70 you already set the username correctly when using the "Take" functionality. For the case where the user fills in the information themselves this is not needed because they are required to only fill in the email. Let me know what you think.

Comment (florinbilt) [reply ui/perfherder/alerts/Assignee.jsx:37]: This line is necessary when we want to reassign an alert. If that line doesn't exist, when enter in edit mode on an already assigned alert, we will see 'mozilla-ldap/fbilt@mozilla.com' instead of 'fbilt@mozilla.com'

File: ui/perfherder/alerts/Assignee.jsx (hunk 1 above)
Comment (florinbilt) [reply ui/perfherder/alerts/Assignee.jsx:49]: I moved this line (event.preventDefault();) inside the if statement because it was also preventing the onchange event on the input field, and that's why the input field was not editable. However, we still need that line because inside the if statement, 'const { failureStatus } = await updateAssignee(newAssigneeUsername);' is called. This line at the end of the function will try to redirect you to '/alerts' regardless of whether the assignment was successful or not.

**Review by beatrice-acasandrei — APPROVED** [review]

File: ui/perfherder/alerts/Assignee.jsx (hunk 1 above)
Comment (beatrice-acasandrei) [reply ui/perfherder/alerts/Assignee.jsx:49]: Thanks for the explanation

**Review by beatrice-acasandrei — APPROVED** [review]
